provisioning/containers/tigervnc/Dockerfile

```
@@ -0,0 +1,32 @@
+FROM ubuntu:20.04
+
+ARG USER=vncuser
```

```suggestion
ARG USER=crownlabs
```

If the name doesn't matter, I would go with crownlabs